shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/impl/EncryptProjectionTokenGenerator.java

```diff
@@ -63,10 +74,22 @@ protected boolean isGenerateSQLTokenForEncrypt(final SQLStatementContext sqlStat
     @Override
     public Collection<SubstitutableColumnNameToken> generateSQLTokens(final SelectStatementContext selectStatementContext) {
         ProjectionsSegment projectionsSegment = selectStatementContext.getSqlStatement().getProjections();
-        // TODO process multiple tables
-        String tableName = selectStatementContext.getAllTables().iterator().next().getTableName().getIdentifier().getValue();
-        return getEncryptRule().findEncryptTable(tableName).map(
-            encryptTable -> generateSQLTokens(projectionsSegment, tableName, selectStatementContext, encryptTable)).orElseGet(Collections::emptyList);
+        Collection<SimpleTableSegment> simpleTableSegments = selectStatementContext.getAllTables();
+        Map<String, SimpleTableSegment> tableSegmentMap = selectStatementContext.getTablesContext().getUniqueTables();
+        List<SubstitutableColumnNameToken> substitutableColumnNameTokenList = new ArrayList<>();
+        for (SimpleTableSegment simpleTableSegment : tableSegmentMap.values()) {
+            String tableName = simpleTableSegment.getTableName().getIdentifier().getValue();
```

Comment:
If we just want to use tableName, we can invoke `tableContext.getTableNames()` rather than `getTablesContext().getUniqueTables()`.